synapse/handlers/e2e_keys.py

```diff
@@ -196,7 +220,73 @@ def do_remote_query(destination):
             ).addErrback(unwrapFirstError)
         )
 
-        return {"device_keys": results, "failures": failures}
+        ret = {"device_keys": results, "failures": failures}
+
+        for key, value in iteritems(cross_signing_keys):
+            ret[key + "_keys"] = value
+
+        return ret
+
+    @defer.inlineCallbacks
+    def query_cross_signing_keys(self, query, from_user_id):
```

Comment:
if we're only getting the keys from the cache, maybe `query_` is not a good name?

Reply:
I just cargo-culted from the `query_local_devices` function below.  What would be a better name?

Reply:
`get_signing_keys_from_cache` ? (there are various other `get_*_from_cache` precedents)